lib/checkleakautovar.h

```diff
@@ -119,8 +119,14 @@ class CPPCHECKLIB CheckLeakAutoVar : public Check {
                     VarInfo *varInfo,
                     std::set<unsigned int> notzero);
 
+    /** Check token inside expression. 
```

Comment:
This doxygen comment is formatted in a weird way. I'd say this is more "compliant" formatting:

    /** Check token inside expression.
     * @param tok token inside expression.
     * @param varInfo Variable info
     * @return next token to process (if no other checks needed for this token). NULL if other checks could be performed.
     */